### DISCUSSION TITLE
Clinvar submission from seqr interface?

### DISCUSSION THREAD

**Discussion by jberghout** [broadinstitute/seqr#4254] [Q&A] [answered]
Is there existing code/system support within seqr for taking in-house interpreted variants and submitting them to ClinVar (ideally via API, but table for export/upload OK too)? 

As an ideal, I'm picturing an automated report workflow that captures the necessary fields into the necessary format, maybe using tags or something like that. But anything that's not starting from scratch would be helpful.

**Comment by hanars** [comment, marked as answer]
So there is a "Submit to Clinvar" tag available in seqr, and you can see all variants tagged with it in the saved variant summary data page: `/summary_data/saved_variants/Submit%20to%20Clinvar`
That page has a "download table" button which allows you to download all the variants with this tag and their relevant fields. You will still need to parse that table to get it formatted for a clinvar submission, but that should not be a very computationally difficult thing to write on your end
